hooks/command

```diff
@@ -219,7 +219,8 @@ target="${BUILDKITE_PLUGIN_DOCKER_ECR_PUBLISH_TARGET:-}"
 enable_buildkit=0
 build_args=()
 caches_from=()
-tags=("${BUILDKITE_BUILD_NUMBER}")
+build_tag="${BUILDKITE_PLUGIN_DOCKER_ECR_PUBLISH_BUILD_TAG:-$BUILDKITE_BUILD_NUMBER}"
```

Comment:
If `build-tag` property is not set, default to the `BUILDKITE_BUILD_NUMBER` value, which was the previous default value